x-pack/plugins/ml/server/routes/README.md

```diff
@@ -2,17 +2,16 @@
 
 This folder contains ML API routes in Kibana.
 
-Each route handler requires [apidoc-markdown](https://github.com/apidoc/apidoc-markdown) annotations in order
-to generate documentation.
+Each route handler requires `summary` and `description` properties for API documentation. Schemas for validation are also used to 
```

Comment:
Are you missing some extra words after `are also used to`? Or can that sentence be removed?

Reply:
fixed in 552556a87e4493f2cd3162602f42e769215b611f